packages/host/app/services/realm-info-service.ts

```diff
@@ -138,4 +138,15 @@ export default class RealmInfoService extends Service {
       waiter.endAsync(token);
     }
   });
+
+  private fetch(url: string, init: RequestInit) {
```

Comment:
`cachedFetch` maybe?

Reply:
Let’s type the return value for this method so it’s clear that it is returning a promise even though it’s a synchronous function.